index.js

```diff
@@ -107,39 +141,89 @@ server.post('/api/actions/:id' , (req , res) => {
     .then(( {id}) => {
```

Comment:
You don't need to use getProjectActions here. You should instead be validating with your middleware.